docs/_docs/monitoring-metrics/new-metrics.adoc

```diff
@@ -280,11 +280,71 @@ Register name: `io.communication`
 [cols="2,1,3",opts="header"]
 |===
 |Name|    Type|    Description
+|ActiveSessionsCount|   integer|   Active TCP sessions count.
 |OutboundMessagesQueueSize|   integer| Outbound messages queue size.
 |SentMessagesCount  | integer| Sent messages count.
 |SentBytesCount | long  |  Sent bytes count.
 |ReceivedBytesCount|  long|    Received bytes count.
 |ReceivedMessagesCount|   integer| Received messages count.
+|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to SSL errors (metric is exported only if SSL is enabled).
+|SslEnabled|   boolean|   Whether SSL is enabled.
+|SslHandshakeDurationHistogram|   histogram|   Histogram of SSL handshake duration in milliseconds (metric is exported only if SSL is enabled).
+|===
+
+
+== Ignite Thin Client Connector
+
+Register name: `connector.client.thin.tcp`
+
+
+[cols="2,1,3",opts="header"]
+|===
+|Name|    Type|    Description
+|ActiveSessionsCount|   integer|   Active TCP sessions count.
+|ReceivedBytesCount|   long|   Received bytes count.
+|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to SSL errors (metric is exported only if SSL is enabled).
+|SentBytesCount|   long|   Sent bytes count.
+|SslEnabled|   boolean|   Whether SSL is enabled.
+|SslHandshakeDurationHistogram|   histogram|   Histogram of SSL handshake duration in milliseconds (metric is exported only if SSL is enabled).
+|===
+
+
+== Ignite REST Client Connector
+
+Register name: `connector.client.rest.tcp`
+
+
+[cols="2,1,3",opts="header"]
+|===
+|Name|    Type|    Description
+|ActiveSessionsCount|   integer|   Active TCP sessions count.
+|ReceivedBytesCount|   long|    Received bytes count.
+|RejectedSslSessionsCount|   integer|   TCP sessions count that were rejected due to SSL errors (metric is exported only if SSL is enabled).
+|SentBytesCount|   long|   Sent bytes count.
+|SslEnabled|   boolean|   Whether SSL is enabled.
+|SslHandshakeDurationHistogram|   histogram|   Histogram of SSL handshake duration in milliseconds (metric is exported only if SSL is enabled).
+|===
+
+
+== Discovery IO
+
+Register name: `io.discovery`
+
+
+[cols="2,1,3",opts="header"]
+|===
+|Name|    Type|    Description
+|CoordinatorSince|   long|   Timestamp since which the local node was the coordinator (metric is only exported from server nodes).
```

Comment:
```suggestion
|CoordinatorSince|   long|   Timestamp since which the local node was the coordinator (metric is exported only from server nodes).
```